tests/integration/strip-data-test-attributes-from-components-test.js

```diff
@@ -0,0 +1,59 @@
+import { moduleForComponent, test } from "ember-qunit";
+import hbs from "htmlbars-inline-precompile";
+
+import config from 'dummy/config/environment';
+
+moduleForComponent('print-test-attributes', 'StripTestSelectorsTransform plugin', {
+  integration: true
+});
+
+if (config.stripTestSelectors) {
+
```

Comment:
This needs a test that while

```hbs
{{my-component data-test-attr='value'}}
```

becomes

```html
<div class="ember-view" data-test-attr="value">
  …
</div>
```

in environments that we don't strip attributes for it becomes

```html
<div class="ember-view">
  …
</div>
```

when `data-test-*` attributes are stripped.